cqf-fhir-cr-cli/src/main/java/org/opencds/cqf/fhir/cr/cli/command/Utilities.java

```diff
@@ -0,0 +1,170 @@
+     */
+    public static String tempConvert(Object value) {
+        if (value == null) {
+            return "null";
+        }
+
+        if (value instanceof Iterable<?> values) {
+            return StreamSupport.stream(values.spliterator(), false)
+                    .map(Utilities::tempConvert)
+                    .collect(Collectors.joining(", ", "[", "]"));
+        }
+
+        if (value instanceof IBaseResource resource) {
+            return resource.fhirType()
+                    + (resource.getIdElement() != null
+                                    && resource.getIdElement().hasIdPart()
+                            ? "(id=" + resource.getIdElement().getIdPart() + ")"
+                            : "");
+        }
+
+        if (value instanceof IBaseDatatype datatype) {
+            return datatype.fhirType();
+        }
+
+        if (value instanceof IBase base) {
+            return base.fhirType();
+        }
+
+        return value.toString();
+    }
+
+    /**
+     * Creates a repository instance based on the provided FHIR context,
+     * terminology URL, and model URL.
+     * @param fhirContext
+     * @param terminologyUrl
+     * @param modelUrl
+     * @return
+     */
+    public static IRepository createRepository(FhirContext fhirContext, String terminologyUrl, String modelUrl) {
+        IRepository data = null;
+        IRepository terminology = null;
+
+        if (modelUrl != null) {
+            Path path = Path.of(modelUrl);
+            data = new IgRepository(fhirContext, path);
+        }
+
+        if (terminologyUrl != null) {
+            terminology = new IgRepository(fhirContext, Path.of(terminologyUrl));
+        }
+
+        return new ProxyRepository(data, null, terminology);
+    }
+
+    /**
+     * Writes a CQL evaluation result to the provided output stream.
+     * Each expression result is written as a line in the format "key=value",
+     * where the key is the expression name and the value is the result of the
+     * expression evaluation, converted to a string using the `tempConvert` method.
+     *
+     * @param result the evaluation result containing expression results
+     * @param outputStream the output stream to write the results to
+     * @throws RuntimeException if an I/O error occurs while writing to the output stream
+     */
+    public static void writeResult(EvaluationResult result, OutputStream outputStream) {
+        try (var writer = new BufferedWriter(new OutputStreamWriter(outputStream, StandardCharsets.UTF_8))) {
+            // Write the result to the output stream
+            for (Map.Entry<String, ExpressionResult> libraryEntry : result.expressionResults.entrySet()) {
+                String key = libraryEntry.getKey();
+                Object value = Utilities.tempConvert(libraryEntry.getValue().value());
+                writer.write(key + "=" + value);
+                writer.newLine();
+            }
+        } catch (IOException e) {
```

Comment:
Can we throw a specific runtimeexception instead, like an InternalErrorException?